### DISCUSSION TITLE
SQL Editor and Active Connections

### DISCUSSION THREAD

**Discussion by bgmxl** [dbeaver/dbeaver#36561] [Q&A] [answered]
Hi，
我在AGS数据库上创建了一个脚本Script-41,这个脚本也适用于AGS-B账套数据库，
但是我在AGS-B上新建SQL编辑器时，就找不到Script-41这个脚本，
可不可以让一个脚本，对应多个数据库？

![image](https://github.com/user-attachments/assets/e43daafe-b830-4a90-bf8e-42dd9efb4ced)

问题好像和#36486 相似。

有没有人可以将我的问题翻译成英文，谢谢。

**Comment by bgmxl** [comment]
I created a script Script-41 on the AGS database, and this script is also applicable to the AGS-B account database.
But when I created a new SQL editor on AGS-B, I couldn't find the script Script-41.
Is it possible to have one script correspond to multiple databases?

**Comment by E1izabeth** [comment, marked as answer]
Hi, @bgmxl 
As far as I can see from your screenshot, the script has not been saved yet. It will appear if you save it.
Also, if you want to see scripts for different connections, no matter which connection is selected in Navigator View, then check this checkbox:

![image](https://github.com/user-attachments/assets/aee5099c-6660-4a33-bf2f-477b2156e544)

**Comment by bgmxl** [comment]
Thanks, @E1izabeth
